locust/render.py

```diff
@@ -144,6 +140,34 @@ def render_json(results: Dict[str, Any]) -> str:
     return json.dumps(results)
 
 
+def render_json_bugout(results: Dict[str, Any]) -> str:
+    summary = results.get("locust")
+    bugout_json = {"summary": summary}
+
+    refs = results.get("refs")
+    if refs is not None:
+        bugout_json.update({"terminal_hash": refs["terminal"]})
+
+    try:
+        # Parse GitHub Action file event.json
+        event_file = "/home/runner/work/_temp/_github_workflow/event.json"
```

Comment:
@kompotkot : I suggest adding this in `ci_helpers/github.py`.

Also, you should read this path from environment variable: `GITHUB_EVENT_PATH` (reference: https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables).

Reply:
Instead of adding it in the `"locust"` object, we can add it at the top-level (same place where we have initial and terminal refs) as a new enricher.